pkg/maker/maker.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package make provides functionality for adding options to components
+package maker
```

Comment:
This package name doesn't tell me what's going on.  `options`?

Reply:
It's a helper library to add options to components. Options seems like a reasonable idea.

Reply:
Been thinking about it some more, love it. I'll refactor based on that idea.